services/mgmt/appplatform/Cargo.toml

```diff
@@ -33,4 +33,3 @@ no-default-version = []
 "package-preview-2021-06" = []
 "package-preview-2020-11" = []
 "package-2020-07" = []
-"package-2019-05-01-preview" = []
```

Comment:
Interesting to see a version removed. What prompted this?

Reply:
The spec was removed. The easiest way to find the changes is to go to its readme and then look at history. https://github.com/Azure/azure-rest-api-specs/blob/main/specification/appplatform/resource-manager/readme.md . That will take you to https://github.com/Azure/azure-rest-api-specs/pull/17506 where it was removed.